sql/moz-fx-data-shared-prod/monitoring_derived/schema_error_counts_v2/schema.yaml

```diff
@@ -27,3 +27,6 @@ fields:
   name: sample_error_messages
   type: STRING
   description: concatenated error messages, limited to 300 characters
+- name: channel
```

Comment:
Per the query, shouldn't `channel` be after `path`?  Or are you intentionally matching the deployed query where new columns get added at the end of the table's schema?

Reply:
This is just auto-generated, but new columns will always get appended to the table.